website/docs/docs/use-dbt-semantic-layer/sl-cache.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Caching in the dbt Semantic Layer"
+id: "sl-cache"
+description: "Cache common queries to speed up performance and reduce query computation."
+tags: [Semantic Layer]
+sidebar_label: "Cache common queries"
+---
+
+
+The dbt Semantic Layer allows you to cache common queries in order to speed up performance and reduce compute on expensive queries. 
+
+There are two layers of caching:
+
+- [**Result caching**](#result-caching) leverages your data platform's built-in caching layer.
+- [**Declarative caching**](#declarative-caching) allows you to pre-warm the cache using saved queries.
+
+## Result caching
```

Comment:
are there any requirements for these two types of caching? (e.g. verions, etc.)

Reply:
requirements:
- dbt cloud team/enterprise
- dbt v 1.6 and higher
- declarative, need to set up 
- need to have job run and prod environment. (will check if for staging)